geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/DestroyIndexFunction.java

```diff
@@ -19,74 +19,65 @@
 import org.apache.geode.cache.Cache;
 import org.apache.geode.cache.CacheClosedException;
 import org.apache.geode.cache.Region;
+import org.apache.geode.cache.configuration.RegionConfig;
 import org.apache.geode.cache.execute.FunctionContext;
 import org.apache.geode.cache.query.Index;
 import org.apache.geode.cache.query.QueryService;
 import org.apache.geode.internal.cache.execute.InternalFunction;
-import org.apache.geode.internal.cache.xmlcache.CacheXml;
-import org.apache.geode.management.internal.cli.domain.IndexInfo;
 import org.apache.geode.management.internal.cli.i18n.CliStrings;
-import org.apache.geode.management.internal.configuration.domain.XmlEntity;
 
 public class DestroyIndexFunction implements InternalFunction {
```

Comment:
Function can extend new `CliFunction` which mandates to return `CliFunctionResult` so that commands can use SingleGfshCommand implementation.